doc/classes/Joint2D.xml

```diff
@@ -74,12 +74,16 @@
 	</methods>
 	<members>
 		<member name="bias" type="float" setter="set_bias" getter="get_bias">
+			The force with which nodes A and B stay in positional relation to each other.
 		</member>
 		<member name="disable_collision" type="bool" setter="set_exclude_nodes_from_collision" getter="get_exclude_nodes_from_collision">
+			Decides, if [member node_a] and [member node_b] should be able to collide.
 		</member>
 		<member name="node_a" type="NodePath" setter="set_node_a" getter="get_node_a">
+			The A node for the joint. Must be of type [PhysicsBody2D].
```

Comment:
To be more accurate, the node's type has to be one that inherits PhysicsBody2D.

    The joint's A node. Must be a [PhysicsBody2D] derived node.